utils/vlog/tests/fasm_features_modes/golden.pb_type.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="fasm_features_modes" num_pb="1">
+  <input name="I0" num_pins="1"/>
+  <input name="I1" num_pins="1"/>
+  <output name="O" num_pins="1"/>
+  <mode name="AND">
+    <pb_type blif_model=".subckt and_gate" name="and_gate" num_pb="1">
+      <xi:include href="../common/and_gate.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
+      <metadata>
+        <meta name="fasm_prefix">THIS_IS_THE_AND_GATE</meta>
+      </metadata>
+    </pb_type>
+    <interconnect>
+      <direct>
+        <port name="I0" type="input"/>
+        <port from="and_gate" name="I0" type="output"/>
+      </direct>
+      <direct>
+        <port name="I1" type="input"/>
+        <port from="and_gate" name="I1" type="output"/>
+      </direct>
+      <direct>
+        <port from="and_gate" name="O" type="input"/>
+        <port name="O" type="output"/>
+      </direct>
+    </interconnect>
+    <metadata>
+      <meta name="fasm_features">COMMON_FASM_FEATURES_FOR_ALL_MODES</meta>
+      <meta name="fasm_prefix">TOP_FASM_PREFIX_FOR_AND</meta>
+    </metadata>
+  </mode>
+  <mode name="XOR">
+    <pb_type blif_model=".subckt xor_gate" name="xor_gate" num_pb="1">
+      <xi:include href="../common/xor_gate.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
+      <metadata>
```

Comment:
Oh, Now I see it does not handle metadata from child. It will not merge it but rather create a second `<metadata>` tag... I have to correct that. But this issue is also relevant to #685

Reply:
@litghost Fixed in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/685